testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -34,11 +37,14 @@
 import itertools
 
 from MDAnalysisTests.plugins.knownfailure import knownfailure
+from MDAnalysisTests import parser_not_found
 
 
 class TestAtom(TestCase):
     """Tests of Atom."""
 
+    @dec.skipif(parser_not_found('DCD'),
```

Comment:
Letting these test fail creates a lot of noise that make nose's report painful to read. Also all the errors generated by these tests are unrelated to what the test tries to asses. The correct semantics would be known failures, but they still display the traceback in the report which left the problem unresolved.

I think the amount of skipped tests is enough to show that there is a problem with the coverage. Also, as soon as the DCD parser works, these tests will not be skipped any more. And so even before we remove the explicit skip decorator.